terraform/stacks/umccr_data_portal/workflow/.terraform.lock.hcl

```diff
@@ -0,0 +1,20 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
Any particular reason why this file should e kept in GH?

Reply:
Yep; according their doco for this deps lock file; similar to `package-lock.json` or `yarn.lock` thing, I reckon.

https://www.terraform.io/docs/language/dependency-lock.html

> You should include this file in your version control repository [...]